code/modules/mob/living/carbon/human/human.dm

```diff
@@ -690,6 +690,10 @@
 	else if (href_list["lookitem"])
 		var/obj/item/I = locate(href_list["lookitem"])
 		usr.examination(I)
+	else if (href_list["listitems"])
+		var/obj/item/I = locate(href_list["listitems"])
```

Comment:
Oh boy href exploits
Admittedly not as bad as some of the others we have, but maybe add a flag or something to the item that allows the contents to be listed this way